### PR TITLE
Add support for mochi-1-preview (genmo) to huggingface.js

### PR DESCRIPTION
**Pull request by ved-genmo**
Add mochi-1-preview to huggingface.js.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:683]: Just a question: isn't this too specific? Should it just be "mochi" or "mochi-1"?

File: packages/tasks/src/model-libraries.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:688]: In [the repo](https://huggingface.co/genmo/mochi-1-preview/tree/main), I see two safetensors files that presumably go together, and no `pt` files. In order to prevent double-counting, would it make sense to count `dit-config.yaml` instead?

**Review by pcuenca — COMMENTED** [review]
Thanks a lot for the contribution (and the release) 🔥

File: packages/tasks/src/model-libraries.ts
Comment (ved-genmo) [reply packages/tasks/src/model-libraries.ts:688]: dit-config.yaml should be removed, ideally we should just count one of the safetensor files.

File: packages/tasks/src/model-libraries.ts
Comment (ved-genmo) [reply packages/tasks/src/model-libraries.ts:683]: I don't have a good understanding of how libraries work (e.g, how is this linked to the actual huggingface page). I named it after the hugging face repository: https://huggingface.co/genmo/mochi-1-preview. But let me know if it should be called something else.

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:683]: It depend how you think the library will evolve. The name should match the library name from the model card metadata ([here](https://huggingface.co/genmo/mochi-1-preview/blob/8e9673c5349979457e515fddd38911df6b4ca07f/README.md?code=true#L8)) which is currently `library_name: mochi-1-preview`. But it doesn't have to match the model repo id. 

If the goal of the Python library on Github is to support this first `preview` version and then maybe some other versions or finetunes, then I would advice to chose `mochi-1` for instance (as @pcuenca suggested). You can see the library as a family of models, not just one. If we go with `mochi-1-preview`, it means that either all other models will have to be tagged as `mochi-1-preview` or that we will have to add support for a new library tag.

File: packages/tasks/src/model-libraries.ts
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:685]: Depending on what's decided above, I would suggest to make this naming more consistent. For instance

```suggestion
		prettyLabel: "Mochi 1",
		repoName: "Mochi 1",
```

File: packages/tasks/src/model-libraries.ts
Comment (pcuenca) [reply packages/tasks/src/model-libraries.ts:683]: Agree with @Wauplin's comments. You can see examples of libraries in this file, they include obvious ones such as `transformers`, but also smaller and more specific ones, for example https://huggingface.co/models?library=depth-anything-v2

Comment (ved-genmo) [reply packages/tasks/src/model-libraries.ts:683]: I see, so this is enabling download counting for an entire library. Which means whatever file path I specify here should be found in all models. To be honest, I don't know if `vae_stats.json` will exist in all possible HuggingFace repositories. But, I guess we can update the elastic search query in the future.

Comment (ved-genmo) [reply packages/tasks/src/model-libraries.ts:683]: Maybe I should rename the library to just "genmo"?

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:683]: Yes! Just `genmo` would be more cleaner and also make this PR forward compatible too.

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:683]: `genmo`'s good in that case!
Since it will be compatible with a large variety of models, it could be nice to have at least a form of similar structure between models. Typically, you could add a `config.json` file with minimal information about the model, its architecture and what it is used for. If all genmo models has one, it might help the library + it will allow us to count the downloads in a consistent way. (I didn't chose the example randomly, `config.json` is actually the default file we are counting downloads from).

But this can be done in a later PR, once you have at least 2 genmo models.

Comment (ved-genmo) [reply packages/tasks/src/model-libraries.ts]: Alright, I updated the repository to have the library "genmo": https://huggingface.co/genmo/mochi-1-preview.

I also updated the library name to genmo.
If there are any small nits, do you want to just do the commits and merge? I'm super un-opinionated here.

**Review by Vaibhavs10 — APPROVED** [review]
Nice! I can see that the original repo also has the same tag: https://huggingface.co/models?other=genmo

We can make the integration better re: download tracking if/ when you release other model checkpoints! Thanks for opening the PR 🤗 

Let's wait for one more LGTM and then we can merge!

**Review by Wauplin — APPROVED** [review]
one more LGTM! Thanks @ved-genmo :hugs:

**Comment by Wauplin** [comment]
I merged the PR. You can expect to see it deployed in the next few days.

**Pull request merged**
